torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2288,6 +2288,23 @@ def sample_inputs_outer(op_info, device, dtype, requires_grad, **kwargs):
     inputs.append(SampleInput(arg_a, args=(arg_b,)))
     return inputs
 
+
+def sample_inputs_igamma_igammac(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
Should this extend

https://github.com/pytorch/pytorch/blob/eea52b7d471350df0cd3823e10b4add5031f35c4/torch/testing/_internal/common_methods_invocations.py#L1260

?

Maybe it would need to add new kwargs like "exclude_scalars" and allowing passing "low" and "high" for make_tensor()?

Reply:
* `exclude_scalars` makes sense.
* For `low` and `high` should it be `low_{lhs/rhs}` and `high_{lhs/rhs}` given that both tensors can have different domain (for a hypothetical function). Though this PR will only introduce `low_lhs` and `low_rhs`.
* Also we will need to have `requires_grad_{lhs/rhs}` as currently autograd is not implemented for one of the argument.

Do you think we should pursue this now? I think we can defer to this changes in `sample_inputs_binary_pwise` till we find another function needing this.

Reply:
Gentle Ping @mruberry :). Let me know how you feel about this. Will address all the reviews accordingly. Thanks!

Reply:
You bring up a good point that this may be unnecessarily complicated. How about just adding a TODO that in the future we might want to extend the pwise binary inputs?

cc @pmeier